src/main/java/com/mathworks/ci/RunMatlabTestsBuilder.java

```diff
@@ -491,29 +442,16 @@ public boolean getDefault() {
         public String getFilePath() {
             return null;
         }
-        
     }
-   
+
     
-    public static class Artifact {
-      
-        @DataBoundConstructor
-        public Artifact() {
-            
-        }
-        
-        
-      
-        public void addFilePathArgTo(List<String> inputArgs) {
-            
-        }
-        
-        public String getFilePath() {
-            return null;
-        }
-        
+    public static abstract class Artifact {
+        public abstract void addFilePathArgTo(List<String> inputArgs);
+
+        public abstract String getFilePath();
+
         public boolean getDefault() {

```

Comment:
I think the name getDefault should be revisited. Perhaps getNullability? (sorry I couldn't seem to think of a better name :-) )

Reply:
is it good idea to use isSelected() or selected() ?  i know its more aligned to functional behavior of UI. but i am running short of names either :)